content/guides/accessibility-for-teams/product-management.md

```diff
@@ -0,0 +1,62 @@
+
+If you have project-specific questions, ask your agency’s accessibility team.
+
+## Learn the basics of accessibility and what your team does to make content accessible
+
+### Steps to take
+- Familiarize yourself with the high-level accessibility roles for each member of your team:
+  - [Front end / development]({{< link "/guides/accessibility-for-teams/front-end-development" >}}): Ensure front-end code is written accessibly and conducts manual and automated testing.
+  - [Content design]({{< link "/guides/accessibility-for-teams/content-design" >}}): Ensure content is written in plain language and headings, images, and links are accurately labeled.
+  - [Visual design]({{< link "/guides/accessibility-for-teams/visual-design" >}}): Ensure that designs are accessible, pages are laid out in a logical order, and content meets color contrast requirements.
+  - [UX design]({{< link "/guides/accessibility-for-teams/ux-design" >}}): Ensure that overall experience is built and designed in an accessible fashion, conduct usability testing with people who need accessibility features.
+- Ensure your team members know where to find the accessibility guidelines for their role and follow them from the start of a project. The guides are located here: [front end]({{< link "/guides/accessibility-for-teams/front-end-development" >}}), [content design]({{< link "/guides/accessibility-for-teams/content-design" >}}), and [visual design]({{< link "/guides/accessibility-for-teams/visual-design" >}}).
+- Take an Accessibility 101 class, such as [Udacity’s Web Accessibility class](https://www.udacity.com/course/web-accessibility--ud891), to get a baseline knowledge of accessibility.
+- Learn how to [navigate a webpage using only your keyboard]({{< link "/guides/accessibility-for-teams/visual-design/#keyboard-access" >}}) and [learn how to use VoiceOver]({{< link "/guides/accessibility-for-teams/visual-design" >}}) so you can spot check new features when necessary.
+
+## Consider how everyone will use the product
+
+### Steps to take
+- Don’t assume that your users don’t have accessibility needs. Even if your product serves a small subset of users, any one individual may experience situational (working in a loud environment) or temporary disabilities (having an arm in a cast), or develop a more permanent one.
+- Consider inclusion in your research and usability testing with people across a range of ages, races, locations, devices, interests, abilities, languages, English proficiency, gender identities, sexual orientations, and access to reliable internet.
+- Consider testing your product with people in their own context, such as people who use alternative reading devices, have color blindness impairments, or motor impairments.
+- Set a regular cadence for testing accessibility scenarios.
+
+#### Resources
+- [Personas for Accessible UX](https://prod.rm.gfolkdev.net/a-web-for-everyone/personas-for-accessible-ux/)
```

Comment:
@bonnieAcameron @ToniBonittoGSA We'll need to review these resource sections if they still are relevant or apply

Reply:
Fixed